pages/posts/stability.md

```diff
@@ -0,0 +1,128 @@
+for continuous evolution without sacrificing interoperability and longevity.
+That's why we've chosen to base our new process on the process that is used to
+evolve the JavaScript language. In our next release, most of the keywords and
+features you use today will be declared stable and they will never change in a
+backward incompatible way again. Features and keywords we aren't comfortable
+making stable yet will become part of a new staged release process that we are
+working on defining now. The goal of the staged release process is to ensure
+that the feature gets sufficient implementation, testing, and real world vetting
+for us to be confident in declaring it stable. The process should not only make
+us more confident, but also allow us to achieve that confidence much more
+quickly.
+
+## Standards considerations
+
+Starting with the next release, the JSON Schema specification will be
+self-published on our website.
+
+One of the concerns about self-publishing is about other standards being able to
+reference the JSON Schema specification. We have received
+[feedback](https://github.com/json-schema-org/json-schema-spec/pull/1277#issuecomment-1228734352)
+from people involved in standards development that our approach would be
+acceptable for them to reference our specification in their standards based on
+our membership with the OpenJS Foundation. We don't know if _all_ standards
+bodies will come to the same conclusion, but this feedback gives us confidence
+that it isn't going to be a significant problem.
+
+Although the main specification will be self published, we are continuing to
+work through the IETF process where it makes sense. For example, we are in the
+process of registering our media types such as `application/schema+json` through
+the HTTPAPIs WG. We are also looking into standardizing reusable components such
+as Relative JSON Pointer through IETF.
+
+## The bottom line
+
+Details about the new process will be shared in a separate post when they are
+finalized, but here are some outcomes users can expect.
+
+* If you use only stable features, you are guaranteed interoperability between
+  JSON Schema libraries and you will never need to update your schemas just for
+  the sake of keeping up with new releases.
+* You can safely use new features before they are stable, as long as the
+  libraries consuming your schemas support the feature.
+* Compatibility/Interoperability guarantees will only apply to the next release
+  and beyond. You will need to update your schemas to the stable version, but
+  you won't have to continue to update them as JSON Schema evolves.
+* Custom dialects and vocabularies will continue to be central concepts for
+  customizing and extending JSON Schema.
+* Implementations only need to support the latest release. A library that
+  supports the 2025 release will automatically support the 2023 and 2024
+  releases. The older releases will no longer need to be maintained as distinct
+  versions.
```

Comment:
I'm wondering, do we need to spell out that this is, from that point forward, not including all previous versions of JSON Schema?
It's implied when you mention 2023, but do we want to avoid people potentially missing that implication?

Reply:
Good point. That could be easily misinterpreted.